project_recalculate/models/project_task.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning, ValidationError
+from datetime import datetime
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    from_days = fields.Integer(
+        string='From days',
+        help='Anticipation days from date begin or date end', default=0)
+    estimated_days = fields.Integer(
```

Comment:
planned_hours means: How many hours is going to consume that task.
estimated_days means: How many days we need to have this task end using a concrete work load.
For example:
If a task is going to be developted by two persons working full-time (8 hours/day) then we have:
- planned_hours: 32 hours
- estimated_days: 2 days

But if this task is developed by one person woring half-time (4 hours/day) we have:
- planned_hours: 32 hours
- estimated_days: 8 days

Here we need that user sets estimated_days because we don't have a work load field. And this is not this addon goal.